### PR TITLE
I4 add units, I1 remove duplicate collection

### PR DESCRIPTION
**Pull request by labradford**
## Related Issues:
 #1 and #4 
Issue 1: Child components with the attribute level="collection" are causing duplicate indexing issues. Added code to indexer to ignore child attributes that have the level="collection" attribute. It is not believed that there are any child components that will need to be indexed as collections. We may need to revisit this at a later time if we find that this is not the case.

Issue 4: IU would like to add the Campus level to indexing. This PR included an EAD with the attribute "mainagencycode" and includes modifications to the ead2_config file to add Campus to the index. The campus is now added to the show page tabs for overview and access. It has also been added as a search facet

Screenshots: 

Duplicate Collections removed:

image

### Indexing Campus:

Access Tab:

image

Facets:

image

Breadcrumbs:

image

### CODE REVIEW

File: app/controllers/catalog_controller.rb [hunk 1]
```diff
@@ -97,7 +97,7 @@ class CatalogController < ApplicationController
     config.add_facet_field 'geogname_sim', label: 'Place', limit: 10
     config.add_facet_field 'places_ssim', label: 'Places', show: false
     config.add_facet_field 'access_subjects_ssim', label: 'Subject', limit: 10
-
+    config.add_facet_field 'campus_unit_sim', label: 'Campus', helper_method: :render_campus_facet
```
Comment (dlim87) [inline app/controllers/catalog_controller.rb]:
```suggestion
    config.add_facet_field 'campus_unit_sim', label: 'Campus', helper_method: :render_campus_facet
    
```
can we add the linebreak back to make the separation clearer?

File: app/models/solr_document.rb [hunk 2]
```diff
@@ -17,4 +17,8 @@ class SolrDocument
   # and Blacklight::Document::SemanticFields#to_semantic_values
   # Recommendation: Use field names from Dublin Core
   use_extension(Blacklight::Document::DublinCore)
+
+  def campus
+    first('campus_unit_ssm')
```
Comment (dlim87) [inline app/models/solr_document.rb]: I doubt there is, but just to double check, there aren't any records that have multiple campuses, right?

Comment (labradford) [reply app/models/solr_document.rb]: no multiple campuses

File: app/controllers/catalog_controller.rb (hunk 1 above)
Comment (labradford) [reply app/controllers/catalog_controller.rb]: I don't understand this suggestion

Comment (dlim87) [reply app/controllers/catalog_controller.rb]: You replaced the one line of whitespace between that set of code and the next one

Comment (dlim87) [reply app/controllers/catalog_controller.rb]: I.e. it went from
```
  config.add_facet_field 'access_subjects_ssim', label: 'Subject', limit: 10

 # Have BL send all facet field names to Solr, which has been the default
```

to 

```
    config.add_facet_field 'access_subjects_ssim', label: 'Subject', limit: 10
    config.add_facet_field 'campus_unit_sim', label: 'Campus', helper_method: :render_campus_facet
    # Have BL send all facet field names to Solr, which has been the default
```

**Pull request merged**